src/components/organisms/Sider/Sider.styled.tsx

```diff
@@ -39,14 +39,14 @@ export const StyledSider = styled(Layout.Sider)<{$isFullScreenLogOutput?: boolea
   }
 `;
 
-export const StyledSiderChildContainer = styled.div`
+export const StyledSiderChildContainer = styled.div<{$width: number}>`
   position: fixed;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
 
   padding-top: 30px;
-  width: 100px;
+  width: ${props => props.$width}px;
```

Comment:
I would consider `zoom` instead, applied via media queries. I don't think there would be any edge cases, but what do you think?

```css
.ant-layout-sider {
  zoom: 0.6;
}

.ant-layout-sider-children > div > div {
  zoom: 1.35;
}
```

Reply:
Haven't seen this approach later, this one is not looking perfectly in mobile